internal/cmd/grants.go

```diff
@@ -278,7 +310,12 @@ func checkResourcesPrivileges(client *api.Client, resource, privilege string) er
 	supportedResources := make(map[string]struct{})
 	supportedRoles := make(map[string]struct{})
 
-	if destination != "infra" {
+	if destination == "infra" {
+		if privilege != "admin" && privilege != "view" {
```

Comment:
This isn't part of the linked issue but was missed in the previous iteration. It checks that only supported roles are specified when creating infra grants

Reply:
We were planning on doing this check in the API instead of the CLI. Should we do that instead of this?

Reply:
In that case, I can omit this change from this PR